### PR TITLE
Do not export all data from front and fixes from questions not answered by everyone

### PR DESCRIPTION
**Pull request by kevin-verschaeve**
questions not answered by everyone questions where not added in the header row

### CODE REVIEW

File: src/Controller/EnMarche/Jecoute/AbstractJecouteController.php [hunk 1]
```diff
@@ -166,7 +166,7 @@ public function jecouteSurveyStatsAction(
         SurveyExporter $exporter
     ): Response {
         if ($format = $request->query->get('export')) {
-            return $exporter->export($survey, $format, true);
+            return $exporter->export($survey, $format, false);
```
Comment (illusionOfParadise) [inline src/Controller/EnMarche/Jecoute/AbstractJecouteController.php:169]: You can not to specify `false`:
```suggestion
            return $exporter->export($survey, $format);
```

File: src/Exporter/SurveyExporter.php [hunk 2]
```diff
@@ -50,41 +51,55 @@ public function export(Survey $survey, string $format, bool $allowedOnly = true)
...
+                if ($fromAdmin) {
+                    $row['Accepte que ses données soient traitées'] = (int) $dataSurvey->getAgreedToTreatPersonalData();
+                }
+
+                $row['Profession'] = $allowPersonalData && $dataSurvey->getProfession() ? $dataSurvey->getProfession() : null;
 
                 /** @var SurveyQuestion $surveyQuestion */
                 foreach ($survey->getQuestions() as $surveyQuestion) {
+                    $questionName = $surveyQuestion->getQuestion()->getContent();
+                    $row[$surveyQuestion->getQuestion()->getContent()] = '';
```
Comment (illusionOfParadise) [inline src/Exporter/SurveyExporter.php:91]: What about:
```suggestion
                    $row[$questionName] = '';
```
?

File: src/Controller/EnMarche/Jecoute/AbstractJecouteController.php (hunk 1 above)
Comment (kevin-verschaeve) [reply src/Controller/EnMarche/Jecoute/AbstractJecouteController.php:169]: yes but, in some cases, like that, I prefer to precise it, so we know that it is intentional and that we really want this behaviour

**Review by ottaviano — APPROVED** [review]

File: src/Exporter/SurveyExporter.php (hunk 2 above)
Comment (kevin-verschaeve) [reply src/Exporter/SurveyExporter.php:91]: yes good catch, I created a variable for that, and forgot to use it !

**Pull request merged**